migrate_cpp/cpp_refactoring/BUILD

```diff
@@ -44,6 +44,43 @@ cc_library(
     ],
 )
 
+cc_library(
```

Comment:
Regarding directories, you might want to either place this code directly under `migrate_cpp`, or make a separate folder -- I don't know what the right choice is. The main intent in naming is that `migrate_cpp` should be the core runner, and `cpp_refactoring` was going to contain a bunch of individual matchers. However, I think with the recursive approach, you're going to end up with different setup. Having just a `rewriter` subdirectory may make sense, or no subdirectory at all.

Reply:
I've moved everything up a level. I don't have strong feelings here and to be honest, I expect the directory structure to be somewhat emergent. I think having everything in the main subdirectory for now is a good option.